pydust/src/types/code.zig

```diff
@@ -0,0 +1,43 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//         http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const std = @import("std");
+const py = @import("../pydust.zig");
+const PyObjectMixin = @import("./obj.zig").PyObjectMixin;
+
+const ffi = py.ffi;
+
+/// Wrapper for Python PyCode.
+/// See: https://docs.python.org/3/c-api/code.html
+pub const PyCode = extern struct {
```

Comment:
you can find a handy reference of the field names in https://docs.python.org/3/library/inspect.html